### ISSUE TITLE
Need a suggestion

### ISSUE THREAD

**Issue by rprasanth** [kraj/localedef#3]
Hi @kraj 

As localedef comes with the Glibc, is there any specific reason for maintaining this source?
May I know the reason for the implementation and its use case, please?

**Comment by kraj** [comment]
Yes its derived from localedef from glibc. this is cross compling localedef, which means it can generate the locale files during cross compile. It is used in yocto project and openembedded to build locales during build.
